Cargo.toml

```
@@ -1,7 +1,7 @@
 [package]
 name = "acquire-imaging"
 authors = ["Nathan Clack <nclack@chanzuckerberg.com>"]
-version = "0.3.0-rc5"
+version = "0.3.0"
```

```suggestion
version = "0.3.0-rc6"
```

Lets keep this as rc6 while the test runners aren't completely passing